src/containers/Map/MapMenuView.tsx

```diff
@@ -307,9 +307,11 @@ const styles = StyleSheet.create({
 		justifyContent: 'space-between',
 	},
 	container: {
-		width: screenWidth,
+		width: screenWidth - 17 * 2,
```

Comment:
Let's declare the "17" as a constant up at the top, so that we have an idea what it's for?